### DISCUSSION TITLE
How to deploy checkpoint file to model?

### DISCUSSION THREAD

**Discussion by thunderdruid** [ACEsuit/mace#316] [Q&A] [answered]
Is there any convient way to deploy checkpoint file (.pt file) to production model (.model file) before the end of training. I didn't find any intruction on this

**Comment by ilyes319** [comment, marked as answer]
The easiest is to restart training for the same number of epochs as the checkpoint file.

**Reply by thunderdruid** [reply]
Hi, Thanks for your answer, it works. 
I've also been messing around deepmd and MACE, and an other problem occurs to me is that what is the format and signs of 'stress' and 'virial' in .xyz file? Suppose I want to use ASE reading OUTCAR from VASP, how am I getting virial information? I know I can transfer 'stress' to 'virial' by - stress * volume, but in ASE stresses are stored in order of [xx,yy,zz,yz,xz,xy], so the virial format should be [(xx, xy, xz), (xy, yy, yz), (xz, yz, zz)]?

**Reply by thunderdruid** [reply]
And ASE is using different signs for stress where its negative for compress and positive for tensile, the opposite of VASP. Is MACE fully compatible with the ASE format so I wouldn't worry about the signs?

**Reply by ilyes319** [reply]
yes exactly MACE is fully compatible with ASE so you should stick with this format.

**Reply by thunderdruid** [reply]
Great, many thanks!
